src/xrGame/DiscordRPC.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+class DiscordRPC
+{
+private:
+    int64_t     start_time          {};
+    string64    build_name          {};
+    string64    current_level_name  {};
+    pcstr       current_task_name   {};
+
+public:
+    DiscordRPC      (){ start_time = time(nullptr); };
+    ~DiscordRPC     (){};
+
+    void Init       ();
+    void Update     (pcstr  level_name = nullptr);
+    void SetTask    (pcstr  task_name);
+    void Deinit     ();
```

Comment:
```suggestion
    void Init();
    void Deinit();

    void Update(pcstr level_name = nullptr);
    void SetTask(pcstr task_name);
```

Reply:
So I have to move Init() functions to constructor, may I delete function? There's no need of it else

Reply:
> So I have to move Init() functions to constructor

No

Reply:
Both constructor and Init stay the same, there's just one line of code that should be moved from `Init` to constructor